library/src/main/java/com/leinardi/android/speeddial/SpeedDialView.java

```diff
@@ -1014,9 +1016,9 @@ public SnackbarBehavior() {
         public SnackbarBehavior(Context context, AttributeSet attrs) {
             super(context, attrs);
             TypedArray a = context.obtainStyledAttributes(attrs,
-                    android.support.design.R.styleable.FloatingActionButton_Behavior_Layout);
+                    R.styleable.FloatingActionButton_Behavior_Layout);
             mAutoHideEnabled = a.getBoolean(
-                    android.support.design.R.styleable.FloatingActionButton_Behavior_Layout_behavior_autoHide,
+                    R.styleable.FloatingActionButton_Behavior_Layout_behavior_autoHide,
```

Comment:
**Reporter**: Android Lint
**Rule**: PrivateResource
**Severity**: WARN
**File**: /home/travis/build/leinardi/FloatingActionButtonSpeedDial/library/src/main/java/com/leinardi/android/speeddial/SpeedDialView.java L1021

Using private resources&#10;The resource &#96;@styleable/FloatingActionButton_Behavior_Layout_behavior_autoHide&#96; is marked as private in com.google.android.material:material&#10;Private resources should not be referenced; the may not be present everywhere, and even where they are they may disappear without notice.&amp;#xA;&amp;#xA;To fix this, copy the resource into your own project instead.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-286141959>*